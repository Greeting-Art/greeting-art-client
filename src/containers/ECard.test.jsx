/**
 * @jest-environment jsdom
 */
import React from 'react';

describe('email setup page for sending eCard', () => {
  it('if test email was sent return 200', async () => {
    
    const sendMsg = {
      method: 'POST',
        headers: {
            'Access-Control-Allow-Origin': '*',
            'Accept': '*/*',
            'Accept-Encoding': 'gzip, deflate, br',
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({
            'to': `test@test.com`,
            'from': 'GreetingArt@outlook.com',
            'subject': `test has send you a Greeting Art`,
            'html': '<h1>hello there</h1>',
            'mail_settings': {
              'sandbox_mode': {
                'enable': true
              }
            }
    })
  }

    const res = await fetch('https://limitless-everglades-53305.herokuapp.com/send', sendMsg)

    expect(res.status).toEqual(200)

  });
});
