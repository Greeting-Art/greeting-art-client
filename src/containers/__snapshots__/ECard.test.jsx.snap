// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`email setup page for sending eCard displays selected art + inputs + send button 1`] = `
<h2>
  loading...
</h2>
`;
